### PR TITLE
Update Calling UI library version to 1.2.0

### PR DESCRIPTION
**Pull request by pavelprystinka**
## Purpose

* Update Calling UI library version

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull request checklist

This PR has considered:
- [ ] Color Theming
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] View Data Injection
- [ ] Demo App UI/UX update

## How to Test

* Open...
* Click on...

## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### CODE REVIEW

**Comment by jpeng-ms** [comment]
there is a script for this: https://github.com/Azure/communication-ui-library-ios/tree/main/scripts. let me know if you have any issue running it

**Comment by emlynmac** [comment]
Probably don't want the plist file on this PR?

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Azure Communication UI Calling Release History
 
+## 1.2.0 (2023-03-08)
+### New Features
+- Introduced `DebugInfo` to get debug information for `CallComposite` [#446](https://github.com/Azure/communication-ui-library-ios/pull/446), [#476](https://github.com/Azure/communication-ui-library-ios/pull/476), [610](https://github.com/Azure/communication-ui-library-ios/pull/610)
```
Comment (vhuseinova-msft) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md:5]: Just wondering if Changelog agreement was changed as we usually don't copy the description of previous beta releases.

Comment (pavelprystinka) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md:5]: I think it is a new features for GA release, as it is no guaranty that features from beta included in GA

**Comment by pavelprystinka** [comment]
> there is a script for this: https://github.com/Azure/communication-ui-library-ios/tree/main/scripts. and https://github.com/Azure/communication-ui-library-ios/tree/develop/scripts let me know if you have any issue running it

Updated all manually.
Have run it as well, no more changes

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md (hunk 1 above)
Comment (vhuseinova-msft) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md:5]: Mobile UI libraries have had linear releases so far but I think we should have a team discussion/agreement on this


Some of previous examples for the same situation:
`- Updated CallingSDK's version to GA in manual installation guide`

**Review by vhuseinova-msft — APPROVED** [review]

**Review by ShaunaSong — APPROVED** [review]

**Pull request merged**